src/languages/en.js

```diff
@@ -403,6 +403,7 @@ export default {
         linkHasBeenResent: 'Link has been re-sent',
         weSentYouMagicSignInLink: ({login}) => `We've sent a magic sign in link to ${login}. Check your Inbox and your Spam folder and wait 5-10 minutes before trying again.`,
         resendLink: 'Resend link',
+        validationCodeFailedMessage: 'It looks like there was an error with your validation link. Either the validation link has expired, or your account does not exist.',
```

Comment:
Which one is it? The account doesn't exist or the validation link has expired? Is there no way to tell the difference?

Reply:
If there's any way to identify from the API response I can do that.

Reply:
Sorry, to clarify, I don't think the error makes sense. We are showing it here:

https://github.com/Expensify/App/blob/4b4f0ce16f410d82c436b8f9833d60ee30b89a61/src/pages/signin/ResendValidationForm.js#L102-L103

Why tell the user the account does not exist? It must exist based on those two conditions?

Reply:
The other thing we are looking at is if the account "exists but is not yet validated" a.k.a. `isOldUnvalidatedAccount` - but that doesn't mean the account does not exist.